lib/scumblr_tasks/security/github_analyzer.rb

```diff
@@ -386,13 +386,13 @@ def run
           end
 
           if response.nil?
-            @retry_interavl = 0
+            @retry_interval = 0
```

Comment:
This series `@retry_interavl` is an actual bug as opposed to just a typo in a comment or a consistently misspelled api.